### DISCUSSION TITLE
用户怎么去修改自己的密码

### DISCUSSION THREAD

**Discussion by huoshenyingmu** [hhyo/Archery#2036] [General]
我没有找到相关的权限，发现DBA的权限也没有修改的方式

**Comment by nick2wang** [comment]
这里不能改么？
![image](https://user-images.githubusercontent.com/33473924/212278758-18911246-04f5-4a65-bbb8-ccaa9d3c57ed.png)
`(http://your_archery_host/admin/password_change/)`
